src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs

```diff
@@ -72,13 +71,13 @@ public abstract class AveragedLinearTrainer<TTransformer, TModel> : OnlineLinear
 
         private protected abstract class AveragedTrainStateBase : TrainStateBase
         {
-            protected Float Gain;
+            protected float Gain;
 
             protected int NumNoUpdates;
 
             // For computing averaged weights and bias (if needed)
-            protected VBuffer<Float> TotalWeights;
-            protected Float TotalBias;
+            protected VBuffer<float> TotalWeights;
+            protected float TotalBias;
             protected Double NumWeightUpdates;
```

Comment:
>Double [](start = 22, length = 6)

Should these be double instead of Double?

Reply:
Changed to double.